lib/theme_check/shopify_liquid/deprecated_filter.rb

```diff
@@ -17,7 +17,7 @@ def labels
       private
 
       def all
-        @all ||= YAML.load(File.read("#{__dir__}/../../../data/shopify_liquid/deprecated_filters.yml"))
```

Comment:
I could not find any suggested valid alternatives to deprecated filters in `filters.json`, so I decided to continue using the information from `deprecated_filters.yml/json`

Reply:
→ https://github.com/Shopify/theme-liquid-docs/issues/64